### DISCUSSION TITLE
I am failing to understand my MONSTER results

### DISCUSSION THREAD

**Discussion by victortoledo** [netZoo/netZooR#212] [Q&A] [answered]
Dear colleagues,

My name is Victor, I am an MSc student in Psychiatry at the University of Sao Paulo. I am interested in sex differences in  gene networks associated with a specific time window in brain fetal development. Currently, our group is using RNAseq data from this dataset (https://doi.org/10.1186/s13059-018-1567-1) to investigate these networks. However, as I am trying to use MONSTER, the randomized samples are all null.

I think it is very important to say that the variance between groups is quite small. WGCNA analysis demonstrates that there is high preservation between groups, and ALPACA scores are also very small. Could this be the reason of this result? I tried to attach the image below.

Thanks a lot for your time! 

Best regards,
Victor

![9f80b59a-9636-494f-8b0c-e2ecb5c6f79c](https://user-images.githubusercontent.com/12415895/133262201-22b71cba-5243-45c2-96be-762dc5c82e43.png)

**Comment by marouenbg** [comment, marked as answer]
Hi @victortoledo ,

Could you please paste the call to MONSTER?

**Reply by victortoledo** [reply]
Here it is:

monsterRes <- monster(exp, design, motif, nullPerms=100, numMaxCores=1, alphaw=0.5)

**Reply by marouenbg** [reply]
Thank you, how many samples do you have in case and in control?

**Reply by victortoledo** [reply]
I have 50 samples in control and 69 in case. We expected the differences to be subtle, since there has been demonstrated that not many genes are differentially expressed between sexes during brain development. However, the author reported 2700+ genes with difference in expression, so I was expecting to find something using the whole Zoo packages.

**Reply by marouenbg** [reply]
OK, this is reasonable. MONSTER basically tells you which transcription factors are 'differentially involved' in the transition from control to case (health to disease for example). The dTFI plot you have tells you in particular which TFs are more active in transition (red dots). The blue dots represent a random null distribution obtained by resampling the data 100 times. The more difference between red and blue dots for a particular TF, the more significant the involvment of that TF in transition.
You can also plot the dTFI plot with rescale='significance', which will standardize the red distribution by the random blue null distribution. 
I don't see this as a problem anyways, however you can get more values for the null distribution by increasing the nullPerms value. Finally, check if the values of the null distribution are indeed 0 or a very small number.

**Reply by victortoledo** [reply]
I just finished to run another subset of samples and the results are the same. Indeed, the null distribution are very small numbers (and some are 0). I am just worried at how to explain this result because of how low my data variance is. I don't quite understand the methods mathematically, just conceptually, and I am worried about that the fact that the null distribution is so peculiar (contrary to most results I've seem in the tutorials and papers).
Your explanation about the significance helped a lot, I am thinking about using the most significant values as a result.
Thank you for the answer so far!
